crates/starknet/src/error.rs

```diff
@@ -19,6 +19,8 @@ pub enum Error {
     IoError(#[from] std::io::Error),
     #[error("Error when reading file {path}")]
     ReadFileError { source: std::io::Error, path: String },
+    #[error("Format error")]
+    FormatError,
```

Comment:
Where is this used? If not used, should it be deleted?

Reply:
Removed